src/tree/SubscriptionTreeItem.ts

```diff
@@ -99,4 +124,12 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
             }
         }
     }
+
+    private async initPostgresChild(account: Server, databases: Databases): Promise<AzureTreeItem> {
+        const accountKindLabel = getExperienceLabel_postgres(account);
```

Comment:
Do you mean revert back to a single experience.ts file like before with all the cosmos experiences code in it?

Reply:
Yes it should be a single file, but maybe make the name `CosmosDBExperiences.ts` instead of `experience.ts`. The whole point is that "Experience" is a concept unique to Cosmos DB. I don't see us ever using that stuff for Postgres and all of the current code in `PostgresExperiences.ts` is unnecessary